NEWS.md

```diff
@@ -1,3 +1,7 @@
+# OBIC 3.0.2 2024-02-09
+## Fixed
+* Fixes error in validation of B_LU_BRP by updating `crops.obic` [OBI-28]
```

Comment:
Please make changelog more descriptive. For example that you added new brp codes for 2023 and fixed classification grasland met/zonder herinzaai